src/Knp/Component/Pager/Event/Subscriber/Sortable/ArraySubscriber.php

```diff
@@ -25,13 +26,23 @@ class ArraySubscriber implements EventSubscriberInterface
      */
     private $propertyAccessor;
 
-    public function __construct(PropertyAccessorInterface $accessor = null)
+    /**
+     * @var Request
+     */
+    private $request;
+
+    public function __construct(PropertyAccessorInterface $accessor = null, Request $request = null)
```

Comment:
It's `2.0` so a BC break is fine (and I don't know why someone would extend this class).

```php
public function __construct(Request $request, PropertyAccessorInterface $accessor = null)
```

Reply:
Swapped